### ISSUE TITLE
ANEs not unpackaged for debugging when compiling with Animate

### ISSUE THREAD

**Issue by yooni126** [BowlerHatLLC/vscode-as3mxml#503]
I have some ANE in native folder in root of my project 
and my lunch.json is :
```
"name": "Debug",
"type": "swf",
"program": ".\\Silver-app.xml",
"preLaunchTask": "Adobe Animate: compile debug - Silver.fla",
"request": "launch",
"extdir": ".\\native\\"
```
but when I debug my project I see this window
![lunch](https://user-images.githubusercontent.com/3766387/95660854-8b1d6b80-0b37-11eb-8436-8e7cb791eb39.png)

**Comment by joshtynjala** [comment]
You should not need to specify `extdir` in your _launch.json_ file. The ANE files will be unpacked by the extension and this field will be populated automatically. You should only need to set `extdir` in _asconfig.json_.

**Comment by yooni126** [comment]
ok, I set it in asconfig.json but unfortunately, it was not resolved 

```
{
	"type": "app",
	"config": "airmobile",
	"airOptions": {
		"extdir": [
			".\\native\\"
		]
	},
	"compilerOptions": {
		"external-library-path": [
			".\\native\\"
		],
		"source-path": [
			"E:\\MyProject\\M.Team\\Tools\\SaffronCode\\",
			"E:\\MyProject\\M.Team\\Tools\\ASPack",
			"E:\\MyProject\\M.Team\\Tools\\SaffronCode",
			"E:\\MyProject\\M.Team\\Tools\\MteamGoogleMapAPI\\src"
		]
	},
	"animateOptions": {
		"file": "Silver.fla"
	},
	"files": [
		"Silver.as"
	]
}
```

**Comment by joshtynjala** [comment]
1) The folder _native_ is inside the same parent folder as _Silver.fla_ and _Silver.as_, correct?
2) Try `"native"` instead of `".\\native\\"`.

**Comment by joshtynjala** [comment]
3) After you set it in _asconfig.json_, did you remove it from _launch.json_?

**Comment by yooni126** [comment]
yes, I don't have any issue when I compile it for mobile but for debugging in Vscode I have this issue
 this is my asconfig.json
```
{
	"type": "app",
	"config": "airmobile",
	"airOptions": {
		"extdir": [
			"native"
		]
	},
	"compilerOptions": {
		"external-library-path": [
			"native",
			"E:\\MyProject\\M.Team\\Tools\\FlexLibs-andOthers",
			"E:\\MyProject\\M.Team\\Tools\\MteamGoogleMapAPI\\src"
		],
		"source-path": [
			"E:\\MyProject\\M.Team\\Tools\\SaffronCode\\",
			"E:\\MyProject\\M.Team\\Tools\\ASPack",
			"E:\\MyProject\\M.Team\\Tools\\SaffronCode",
			"E:\\MyProject\\M.Team\\Tools\\MteamGoogleMapAPI\\src"
		]
	},
	"animateOptions": {
		"file": "Silver.fla"
	},
	"files": [
		"Silver.as"
	]
}
```
this is my lunch.json
```
{
	"configurations": [
		{
			"name": "Debug",
			"type": "swf",
			"program": ".\\Silver-app.xml",
			"preLaunchTask": "Adobe Animate: compile debug - Silver.fla",
			"request": "launch",
		}
	],
	"version": "0.2.0"
}
```
![folder](https://user-images.githubusercontent.com/3766387/95778143-57c41380-0cd4-11eb-8ec8-07f4756b60ec.png)

**Comment by joshtynjala** [comment]
I've determined that the extension is not preparing the ANEs for debugging when building with Animate. I'll get that fixed.

**Comment by yooni126** [comment]
Hi josh
thank you for your effort 
But the problem is still unresolved
my Vscode version is 1.50.1 and MXML version is 1.3.0
also, I remove this command in lunch.json
`"preLaunchTask": "Adobe Animate: compile debug - Silver.fla",`

**Comment by joshtynjala** [comment]
You must wait for me to release a new version.

**Comment by joshtynjala** [comment]
I just released 1.4.0, including this fix.
